extensions/interactions/AlgebraicExpressionInput/directives/oppia-interactive-algebraic-expression-input.component.spec.ts

```diff
@@ -94,7 +94,7 @@ describe('AlgebraicExpressionInputInteractive', () => {
 
   beforeEach(() => {
     windowRef = TestBed.inject(WindowRef);
-    windowRef.nativeWindow.Guppy = MockGuppy;
+    windowRef.nativeWindow.Guppy = MockGuppy as unknown as Guppy;
```

Comment:
Please add a TODO for the removal of the `unknown`

Reply:
Added a comment and files an issue.